docs/oracle/tech-architecture.md

```diff
@@ -3,72 +3,44 @@ title: Technical Architecture of the DVM
 sidebar_label: Technical Architecture
```

Comment:
This page reminds me, can you also add an Optimistic Oracle definition to Glossary?

Reply:
Added an Optimistic Oracle definition. I want to do more work on the glossary in the next PR.